app/src/main/res/values/dimens.xml

```diff
@@ -17,4 +17,8 @@
     <dimen name="sp_15">15sp</dimen>
     <dimen name="dp_30">30dp</dimen>
     <dimen name="sp_25">25sp</dimen>
+    <dimen name="dp_320">320dp</dimen>
+    <dimen name="dp_220">220dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_220` appears to be unused
 Unused resources make applications larger and slow down builds.

Reply:
2 cái size này em dùng sau ạ